.gitignore

```diff
@@ -2,6 +2,9 @@
 frontend/dist/bundle.js
 build
 target/
+# Packr generated files
+packrd/
+*-packr.go
```

Comment:
This would break the integration with the operator. Packr files need to be checked in to be picked up when using the project as a library.